libsoftwarecontainer/component-test/softwarecontainerlib_componenttest.cpp

```diff
@@ -487,7 +501,7 @@ TEST_F(SoftwareContainerApp, TestUnixSocket) {
 
                 close(fd2);
 
-                return done == 1 ? EXISTENT : NON_EXISTENT;
+                return done == 1 ? SUCCESS : FAILURE;
```

Comment:
Imo it makes more sense to return known values than relying on that they will match what we get as value for `done`